src/main/java/at/hannibal2/skyhanni/config/features/garden/visitor/ShoppingListConfig.java

```diff
@@ -26,13 +26,13 @@ public class ShoppingListConfig {
     public boolean onlyWhenClose = false;
 
     @Expose
-    @ConfigOption(name = "Bazaar Alley", desc = "Show the Visitor Items List while inside the Bazaar Alley in the Hub. " +
+    @ConfigOption(name = "Bazaar Alley", desc = "Show the Visitor Items List while inside the Bazaar Alley in the Hub.\n" +
         "This helps buying the correct amount when not having a Booster Cookie Buff active.")
```

Comment:
here again, do we want use yellow color for the last line, or remove the line break?